hack/lib/golang.sh

```diff
@@ -780,6 +780,20 @@ kube::golang::build_binaries_for_platform() {
     testpkg=$(dirname "${test}")
 
     mkdir -p "$(dirname "${outfile}")"
+
+    # TODO(saschagrunert): try to remove once go1.20.1 got released, watch
+    # out for the release notes which may fix relocation issues like:
+    #
+    # /tmp.k8s/go-link-865684912/go.o: in function `k8s.io/kubernetes/vendor/github.com/aws/aws-sdk-go/service/ec2.(*ModifyCapacityReservationInput).GoString':
+    # go.go:(.text+0x207ca88): relocation truncated to fit: R_ARM_CALL against `runtime.duffcopy'
+    # go.go:(.text+0x207ca94): relocation truncated to fit: R_ARM_CALL against `runtime.duffcopy'
+    #
+    # Ref: https://github.com/kubernetes/kubernetes/issues/115605
+    if [[ "${test}" == k8s.io/kubernetes/test/e2e_node/e2e_node.test && "${platform}" == linux/arm ]]; then
+      kube::log::info "    disabling CGO for binary: ${test}"
+      CGO_ENABLED=0
```

Comment:
does setting this persist beyond this scope (e.g. does it pollute CGO_ENABLED values for other build calls)?

Reply:
and should this be set only for the invocation of the `go test` call immediately below?

Reply:
Changed as suggested by using a new local variable, which only applies to the `go test` statement.